ios/CozyReactNative/Info.plist

```diff
@@ -37,6 +37,10 @@
 	</array>
 	<key>CFBundleVersion</key>
 	<string>0100011</string>
+	<key>FIREBASE_ANALYTICS_COLLECTION_DEACTIVATED</key>
+	<true/>
+	<key>GOOGLE_ANALYTICS_IDFV_COLLECTION_ENABLED</key>
```

Comment:
The android config has 3 values:
```
firebase_analytics_collection_deactivated
google_analytics_adid_collection_enabled
google_analytics_ssaid_collection_enabled
```
This ios config has only 2 values
```
FIREBASE_ANALYTICS_COLLECTION_DEACTIVATED
GOOGLE_ANALYTICS_IDFV_COLLECTION_ENABLED
```
Isn't `google_analytics_adid_collection_enabled` available on iOS?

Reply:
Yes it is a little different between iOS and Android. The Ad ID 

[More information](https://firebase.google.com/docs/analytics/configure-data-collection?platform=ios#permanently_deactivate_collection_2)